ArduinoCode/Pedal_Potentiometers/Pedal_Potentiometers.ino

```diff
@@ -26,22 +17,30 @@ int brakePin1Val;
 int brakePin2Val;
  
 // CAN stuff
-const int CAN_ACCEL = 0x01;       // device's CAN id
-const int CAN_ACCEL_BROKE = 0x02; // device error ID
-const int CAN_BRAKE = 0x03;       // device's CAN id
-const int CAN_BRAKE_BROKE = 0x04; // device error ID
+const int CAN_BRAKE = 0x03;        // device's CAN id
+const int CAN_MOTOR = 0xC0;        // device's CAN id
+const int CAN_ACCEL_BROKE = 0x02;  // device error ID
```

Comment:
The id 0x02 is already used for the forward/reverse value form the dashboard. Make this 0x05